src/component/ResponsiveContainer.tsx

```diff
@@ -57,6 +58,7 @@ export const ResponsiveContainer = forwardRef(
       id,
       className,
       onResize,
+      style,
```

Comment:
looks like this might need defaulted to `{}` since it is optional? 

I don't think `undefined` can be spread

Reply:
> looks like this might need defaulted to `{}` since it is optional?
> 
> I don't think `undefined` can be spread

Good catch. Fixed!